### ISSUE TITLE
Implement create issue REST API 

### ISSUE THREAD

**Issue by alansemenov** [enonic/xp#4547]
Depends on #4559 
Implement REST API to create an issue, create in a new Resouce IssueResource

Fields:
- title (mandatory)
- list of contentId (mandatory)
- list of approver user ids (mandatory)
- description (optional)

Id, date/time, status should be set automatically. New issues should all get "Open" status by default.

API should respond with JSON of the new issue (if it was created successfully) or error message (if not).
